contrib/completion/hx.nu

```diff
@@ -0,0 +1,25 @@
+# Completions for Helix: <https://github.com/helix-editor/helix>
+#
+# NOTE: the `+N` syntax is not supported in Nushell (https://github.com/nushell/nushell/issues/13418)
+#       so it has not been specified here and will not be proposed in the autocompletion of Nushell.
+#       The help message won't be overriden though, so it will still be present here
+
+def health_categories [] { ["all", "clipboard", "languages"] }
+
+def grammar_categories [] { ["fetch", "build"] }
+
+# A post-modern text editor.
+export extern hx [
+    --help(-h),                                 # Prints help information
+    --tutor,                                    # Loads the tutorial
+    --health: string@health_categories = "all", # Checks for potential errors in editor setup
+    --grammar(-g): string@grammar_categories,   # Fetches or builds tree-sitter grammars listed in `languages.toml`
+    --config(-c): glob,                         # Specifies a file to use for configuration
+    -v,                                         # Increases logging verbosity each use for up to 3 times
+    --log: glob,                                # Specifies a file to use for logging
+    --version(-V),                              # Prints version information
+    --vsplit,                                   # Splits all given files vertically into different windows
+    --hsplit,                                   # Splits all given files horizontally into different windows
+    --working-dir(-w): glob,                    # Specify an initial working directory
```

Comment:
We don't accept globs (if the shell expand the globals there are multiple paths which we don't support and if it doesn't expand then helix won't expand itself). I don't know much about nushell but is it possible to just specify "path but no glob" here?

Reply:
That's the responsibility of the caller to only provide one file IMO. Supporting globs means it's possible to not know the exact path and still type `~/*/my-repo-that-is-only-in-one-subdir`. That's already something that is possible in bash for example